src/cli/build/esbuild.ts

```diff
@@ -46,20 +34,122 @@ export const esbuild = async (
 
   const start = process.hrtime.bigint();
 
-  // TODO: support `bundle`, `minify`, `splitting`, `treeShaking`
-  const bundle = false;
+  switch (mode) {
+    case 'build':
+      await runBuild({
+        compilerOptions,
+        debug,
+        entryPoints,
+        tsconfig: tscArgs.pathname,
+      });
+
+      break;
+
+    case 'build-package':
+      await runBuild({
+        bundle: true,
+        compilerOptions: {
+          ...compilerOptions,
+          module: ModuleKind.ESNext,
+        },
+        debug,
+        entryPoints,
```

Comment:
I think these `entryPoints` are not correct for `bundle: true`. Strictly speaking it should only output the root `lib/index.mjs` unless more `exports` are defined.